### ISSUE TITLE
Remove unused components - to be continued

### ISSUE THREAD

**Issue by Th3da** [DetektivKollektiv/angular-ui#154]
- [ ] app-submit (Alte Version, die aktuelle Version heißt app-submit-page

**Comment by SandroGlueck** [comment]
@Th3da hast du damit schon angefangen? Sonst würde ich das machen.

**Comment by Th3da** [comment]
nein, habe nur gelistet, wenn ich über etwas gestolpert bin - wollte aber ungern selbst löschen.
Ist auch nicht vollständig, zb bei den Auth-Kompenenten gibt es, glaub ich, auch noch einiges doppelt.

Können wir ein Archiv einrichten? Einiges kann sicher klar weg, habe aber auch Teile gesehen, von denen man ggf noch profitieren könnte.
